plugins.yaml

```diff
@@ -340,4 +342,21 @@
   gmodProject: JBrowse
   location: https://github.com/bionanogenomics/JBrowsePlugin/
   license: MIT
-  name: BioNanoGenomics
```

Comment:
@bhofmei, seems like your commit inadvertently deleted a line in the config. Might want to fix this!